src/ctia/task/migrations.clj

```diff
@@ -55,6 +56,98 @@
                (dissoc :target))
            doc))))
 
+;;-- Rename observable type
+
+(defn with-renamed-observable-type
+  [observable old new]
+  (update observable
+          :type
+          (fn [obs-type]
+            (if (= obs-type old)
+              new
+              obs-type))))
+
+(defn with-renamed-observable-types
+  [c old new]
+  (mapv #(with-renamed-observable-type % old new)
+        c))
+
+(defn rename-sighting-relations-observable-types
+  [relation old new]
+  (-> relation
+      (update :source with-renamed-observable-type old new)
+      (update :related with-renamed-observable-type old new)))
+
+(defn rename-sighting-observable-types
+  [sighting old new]
+  (-> sighting
+      (update :observables with-renamed-observable-types old new)
+      (update :relations
+              (fn [relations]
+                (mapv #(rename-sighting-relations-observable-types % old new)
+                      relations)))))
+
+(defn rename-judgement-observable-type
+  [judgement old new]
+  (update judgement
+          :observable
+          with-renamed-observable-type
+          old
+          new))
+
+(defn rename-bundle-observable-types
+  [bundle old new]
+  (-> bundle
+      (update :sightings (fn [sightings]
+                           (mapv #(rename-sighting-observable-types % old new)
+                                 sightings)))
+      (update :judgements (fn [judgements]
+                            (mapv #(rename-judgement-observable-type % old new)
+                                  judgements)))
+      (update :verdicts (fn [verdicts]
+                          (mapv #(rename-judgement-observable-type % old new)
+                                verdicts)))))
+
+(defn rename-casebook-observable-types
+  [{:keys [observables bundle] :as casebook} old new]
+  (cond-> casebook
+    (seq observables) (update :observables with-renamed-observable-types old new)
+    bundle (update :bundle rename-bundle-observable-types old new)))
+
+(defn rename-observable-type
```

Comment:
I would have made a piece of code that given any document traverses it and changes the observable_type key,
but it's a matter of taste.